nautobot_chatops/api/views/lookup.py

```diff
@@ -32,3 +33,26 @@ def get(self, request, *args, **kwargs):
             return HttpResponseNotFound(f"No {request.GET['grant_type']} {request.GET['name']} found")
 
         return JsonResponse(data={"value": value})
+
+
+class UserEmailLookupView(View):
+    """Look up a user_id by email."""
+
+    http_method_names = ["get"]
+
+    def get(self, request, *args, **kwargs):
+        """Handle an inbound GET request for a specific access grant value."""
+        for required_param in ("email", "platform"):
+            if required_param not in request.GET:
+                return HttpResponseBadRequest(f"Missing mandatory parameter {required_param}")
```

Comment:
```suggestion
        missing_params = set(("email", "platform")) - set(request.GET.keys())
        if missing_params:
            return HttpResponseBadRequest(f"Missing mandatory parameter {missing_params}")
```

The new version notifies user about both missing params.